### DISCUSSION TITLE
6.2 required for 2.0+?

### DISCUSSION THREAD

**Discussion by tayclark** [taskforcesh/bullmq#2016] [General]
I noticed in 2.0.0 release notes that minimum of 6.2 is _recommened._ and I was wondering if there was any specific reason for that

**Comment by manast** [comment]
Here is the explanation: https://blog.taskforce.sh/bullmq-2-0/

"We have raised the minimum version of Redis to 6.2. Although version 5.0+ will kind of work, it is highly recommended that you upgrade to at least version 6, since with the removal of the QueueScheduler we now need that the timeout argument is interpreted as a "double" in the [BRPOPLPUSH](https://redis.io/commands/brpoplpush/?ref=blog.taskforce.sh) command. If you are still on 5.0 version, the delayed jobs will sometimes be delayed more than they should as the timeout cannot represent exactly the remaining delay to the next job."
